static/app/views/profiling/profileSummary/regressedProfileFunctions.tsx

```diff
@@ -162,8 +162,9 @@ export function MostRegressedProfileFunctions(props: MostRegressedProfileFunctio
 
   const onChangeTrendType = useCallback(v => setTrendType(v.value), []);
 
-  const hasDifferentialFlamegraphPageFeature =
-    false && organization.features.includes('profiling-differential-flamegraph-page');
+  const hasDifferentialFlamegraphPageFeature = organization.features.includes(
+    'profiling-differential-flamegraph-page'
+  );
```

Comment:
cc @JonasBa @Zylphrex you're the only two mentioned in the flagpole config for this. It'll be enabled now for ya.